src/testutils/mockNetworkProvider.ts

```diff
@@ -216,11 +232,6 @@ export class MockNetworkProvider implements INetworkProvider {
         throw new ErrMock("Transaction not found");
     }
 
-    async getTransactionStatus(txHash: string): Promise<TransactionStatus> {
```

Comment:
Completely removed? Maybe we should still keep it & add it in the specs?

https://github.com/multiversx/mx-sdk-specs/blob/main/network-providers/interface.md